docs/system-administration/ubiquiti.md

```diff
@@ -0,0 +1,18 @@
+# Ubiquiti
+
+This page contains information relating to the Ubiquiti line of products.
+
+## Adopt a new Access Point with remote Unifi controller software
+
+If your Unifi controller software doesn't run on your LAN it is neccessary to tell the AP where it is. This allows the AP to be 'adopted' by the controller and have its configuration managed by it.
+
+To do this you will need to SSH to the AP (default user and password is `ubnt`).
+
+    ssh <ip-of-access-point>
+    set-inform http://<controller-url>:8080/inform
```

Comment:
Possibly tangential, super nitpick comment: Is the controller connection really over non-TLS?

Reply:
yes, it is http. it's only the announce URL though, the controller itself is TLS via whatever means you pick. i could go into that but i'd say it's not really in scope for this article unless we write another and link to it. i wasn't too keen on that plan for now though #abitlazy